core/tests/puppeteer-acceptance-tests/spec/practice-question-admin-tests/remove-contribution-rights.spec.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Acceptance Test for question admins to remove reviewing
+ * question rights to users.
+ */
+
+const userFactory = require(
+  '../../puppeteer-testing-utilities/user-factory.js');
+const testConstants = require(
+  '../../puppeteer-testing-utilities/test-constants.js');
+
+const DEFAULT_SPEC_TIMEOUT = testConstants.DEFAULT_SPEC_TIMEOUT;
+
+describe('Question Admin', function() {
+  let questionAdmin = null;
+
+  beforeAll(async function() {
+    questionAdmin =
+      await userFactory.createNewQuestionAdmin('questionAdm');
+  }, DEFAULT_SPEC_TIMEOUT);
+
+  it('should be able to provide rights to review and submit questions to user.',
```

Comment:
I think you might as well combine this with the test in the previous file (and change its name to add-and-remove-contribution-rights) so that we don't have to bear the setup cost twice, especially since this can quite easily follow on from the previous test. What do you think?

Reply:
OK, please go ahead and combine them. (Also, in future, please do apply editorial discretion to the journeys given in the spreadsheet -- if you think things would be better combined or separated, please ask on the groupchat and we can give you feedback on that. Thanks!)

Reply:
Done. The files are combined.